main.tf

```diff
@@ -8,7 +8,7 @@ module "lambda" {
   function_name = var.name
   description   = "Gather open pull requests and send it to slack"
   handler       = "main.lambda_handler"
-  runtime       = "python3.6"
+  runtime       = var.lambda.runtime
   timeout       = 30
```

Comment:
Nit but since we have an object for this, maybe we can add the timeout, function_name, dry_run, to the lambda object

Reply:
Definitely not for `name` or `dry_run`, that would be backwards-incompatible.

Regardless, we'll be ripping this out to replace it with the newer, still-maintained lambda module soonish. This is just trying to address the runtime since it's a blocker for deploying this in the immediate-term.